.env.example

```diff
@@ -0,0 +1,3 @@
+HOST_PORT=8000                        # Specifies the port in which the app will be exposed (when ran using Docker)
+MONGO_URI=mongodb://localhost:27017   # Specifies the URI for an external mongo database (OPTIONAL)
+PORT=8000                             # Specifies the port in which the app will be exposed (OPTIONAL, when ran without using Docker)
```

Comment:
The difference between `HOST_PORT` and `PORT` needs clarification. One is for node, another is for docker.

Maybe the names can be changed or the variables merged (docker can pass a default port to node and remap it externally, if this is preferred), but at least make the explanation actually be different 😅